opentracing/harness/api_check.py

```diff
@@ -39,54 +43,58 @@ def check_trace_attribute_values(self):
         """
         return True
 
-    def test_start_trace(self):
+    def test_start_span(self):
         tracer = self.tracer()
-        span = tracer.start_trace(operation_name='Fry')
+        span = tracer.start_span(operation_name='Fry')
         span.finish()
-        with tracer.start_trace(operation_name='Fry',
-                                tags={'birthday': 'August 14 1974'}) as span:
+        with tracer.start_span(operation_name='Fry',
+                               tags={'birthday': 'August 14 1974'}) as span:
             span.log_event('birthplace',
-		           payload={'hospital': 'Brooklyn Pre-Med Hospital',
+                           payload={'hospital': 'Brooklyn Pre-Med Hospital',
                                     'city': 'Old New York'})
-        tracer.close()
+        tracer.flush()
 
-    def test_join_trace(self):
+    def test_start_span_with_parent(self):
         tracer = self.tracer()
-        parent_span = tracer.start_trace(operation_name='parent')
+        parent_span = tracer.start_span(operation_name='parent')
         assert parent_span is not None
-        span = tracer.join_trace(operation_name='Leela',
-                                 parent_span=parent_span)
+        span = tracer.start_span(operation_name='Leela',
+                                 parent=parent_span)
         span.finish()
-        span = tracer.join_trace(operation_name='Leela',
-                                 parent_span=parent_span,
+        span = tracer.start_span(operation_name='Leela',
+                                 parent=parent_span,
                                  tags={'birthplace': 'sewers'})
         span.finish()
-        tracer.close()
+        parent_span.finish()
+        tracer.flush()
+
+    def test_start_child_span(self):
+        tracer = self.tracer()
+        parent_span = tracer.start_span(operation_name='parent')
+        assert parent_span is not None
+        child_span = span.start_child_span(parent_span, operation_name='Leela')
```

Comment:
it might be better to import `start_child_span` (and any other helper methods) into `__init__.py` so that people can used it as `opentracing.start_child_span()`. I would rather have `span` name to be left to local variables. Not that it's impossible to rename during import, but this would make life easier for people, I think, if they can access all of OT via a single `opentracing` import / namespace.